linter/rules/average_measure_name_must_start_with_avg_or_average.py

```diff
@@ -1,16 +1,12 @@
 from linter.rule import Rule
 from typing import Any, Tuple
-from linter.helpers import check_field_naming_convention_by_type
 
 
 class AverageMeasureNameMustStartWithAvgOrAverage(Rule):
     def applies_to() -> Tuple[str, ...]:
         return ('measure',)
 
     def run(self, field: Any) -> bool:
-        field_name = field['name']
-        type = field.get('type')
-        check_field_type = ['average', 'average_distinct']
-        field_name_must_start_with = ['avg','average']
-
-        return check_field_naming_convention_by_type(field_name, type, check_field_type, field_name_must_start_with)
+        if field.get('type') in ['average', 'average_distinct']:
+            return field.get('name').startswith('avg_') or field.get('name').startswith('average_')
```

Comment:
The `startswith` method accepts tuples, so you can do:

```
.startswith('avg_', 'average_')
```

to keep it a little more concise.

Reply:
Ah, good call out. Updated 👍🏾